src/components/Todos/DoinItGrid.js

```diff
@@ -0,0 +1,22 @@
+import { TodoList } from '../UI/TodoCard/TodoList.styled';
+import { TodoLi } from '../UI/TodoCard/TodoListItem.styled';
+import { Container } from '../UI/Grid/Container';
+import Todos from './Todos';
+import { ListHeadline } from '../UI/TodoCard/ListHeadline.styled';
+
+export default function DoinItGrid({ todos }) {
+	return (
+		<Container>
+			<ListHeadline>DoingIT</ListHeadline>
+			<TodoList role="list">
+				{todos
+					.filter(todos => todos.status === 'doingIt')
+					.map(todos => (
+						<TodoLi key={todos.id}>
+							<Todos title={todos.title} />
```

Comment:
Same here.
Besides, the `Todos` component only contains a single Todo, right? I would thus name it with singular as well.
```suggestion
					.map(todo => (
						<TodoLi key={todo.id}>
							<Todo title={todo.title} />
```